File: PHP/Iniciante/1009.php
<?php

$nome = readline();
$sal_fixo = floatval(readline());
$valor_vendas = floatval(readline());

$salario = number_format($sal_fixo + ($valor_vendas * 0.15), 2, '.', '');

echo "TOTAL = R$ $salario\n"

?>

File: GO/Iniciante/1009.go
package main

import (
	"fmt"
)

func main() {

  var nome string
  var salario float64
  var valor_vendas float64

  fmt.Scan(&nome)
  fmt.Scan(&salario)
  fmt.Scan(&valor_vendas)

  salario += valor_vendas * 0.15

  fmt.Printf("TOTAL = R$ %.2f\n", salario)
  
}
